java/.ci/Jenkinsfile

```diff
@@ -769,7 +769,29 @@ pipeline {
                     }
                 }
             }
-            
+            stage('Parallel Execution 9'){
+                parallel{
+                    stage('PubSubLite To GCS') {
+                        steps{
+                            sh '''
+                                export GCP_PROJECT=yadavaja-sandbox
+                                export REGION=us-west1
+                                export GCS_STAGING_LOCATION="gs://python-dataproc-templates-temp"
+
+                                cd java
+
+                                bin/start.sh \
+                                -- --template PUBSUBLITETOGCS \
+                                --templateProperty pubsublite.to.gcs.input.subscription.url=projects/$GCP_PROJECT/locations/$REGION/subscriptions/psltobt-sub \
+                                --templateProperty pubsublite.to.gcs.processing.time.seconds=1 \
+                                --templateProperty pubsublite.to.gcs.timeout.ms=120000 \
+                                --templateProperty pubsublite.to.gcs.output.location=gs://saumyasinha/pubsublite/output \
+                                --templateProperty pubsublite.to.gcs.checkpoint.location=gs://saumyasinha/pubsublite/checkpointLocation
```

Comment:
Avoid using your bucket. As a best practice, please use a generic output location similar to other template's int test.